src/Plugin.cpp

```diff
@@ -76,29 +76,34 @@ void Plugin::init() {
 
   // create labels for all bodies that already exist
   for (auto const& body : mSolarSystem->getBodies()) {
-    mAnchorLabels.push_back(std::make_unique<AnchorLabel>(
+    mAnchorLabels.emplace_back(std::make_unique<AnchorLabel>(
```

Comment:
There should be no difference here, since the object creation happens in make_unique. In both cases the move constructer will be called.